dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -274,8 +242,8 @@ protected static String createWorkflowTrsUrl(Workflow workflow, WorkflowVersion
      * @param doiAlias workflow alias constructed using a DOI
      */
     private static void setMetadataRelatedIdentifiers(DepositMetadata depositMetadata,
-            String dockstoreGA4GHBaseUrl, String dockstoreUrl, String workflowUrl,
-            Workflow workflow, WorkflowVersion workflowVersion, String doiAlias) {
+                                                      String dockstoreGA4GHBaseUrl, String dockstoreUrl, String workflowUrl,
+                                                      Workflow workflow, WorkflowVersion workflowVersion, String doiAlias) {
```

Comment:
IMHO, this indentation scheme is very much different than our established scheme, and because consistent formatting is generally a good thing, we should make it more compatible.